package.json

```diff
@@ -12,26 +12,20 @@
     "build": "tsc",
     "watch": "tsc-watch"
   },
-  "files": [
-    "lib/**/*",
-    "fonts/**/*",
-    "package.json",
-    "LICENSE",
-    "README"
-  ],
+  "files": ["lib/**/*", "fonts/**/*", "package.json", "LICENSE", "README"],
   "dependencies": {
     "@sentry/node": "^6.2.0",
     "@types/echarts": "^4.9.3",
     "@types/express": "^4.17.11",
     "@types/joi": "^17.2.3",
-    "@types/node-fetch": "^2.5.8",
     "@types/yargs": "^16.0.0",
+    "abort-controller": "^3.0.0",
     "canvas": "^2.6.1",
     "dotenv": "^8.2.0",
     "echarts": "4.7.0",
     "express": "^4.17.1",
     "joi": "^17.4.0",
-    "node-fetch": "^2.6.1",
+    "node-fetch": "v3.0.0-beta.9",
```

Comment:
This adds better support for AbortController, also apparently it's an important security release? https://github.com/node-fetch/node-fetch/releases/tag/v3.0.0-beta.9

Reply:
Beta dependencies on prod, love it 🤣